PENDING.md

```diff
@@ -38,6 +38,7 @@ BREAKING CHANGES
 * [types] sdk.NewCoin now takes sdk.Int, sdk.NewInt64Coin takes int64
 * [cli] #1551: Officially removed `--name` from CLI commands
 * [cli] Genesis/key creation (`init`) now supports user-provided key passwords
+* [cli] `gaiacli stake edit-validator` now requires a `moniker`, `identity`, `website`, and `details` to be specified
```

Comment:
Mind adding the PR reference here?

Reply:
[#2035](https://github.com/cosmos/cosmos-sdk/pull/2035/files) at the beginning plz!